package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "reactNativeApp",
+  "name": "reactnativeapp",
```

Comment:
Why do you change the name of the package?

Reply:
It was giving me an ugly yellow warning on my IDE 🙈